qutebrowser/browser/hints.py

```diff
@@ -47,6 +54,9 @@
                                    'download', 'userscript', 'spawn'])
 
 
+WordHintingError = Exception
```

Comment:
Please make this a proper exception subclass - your `except WordHintingError as e:` actually is the very same as `except Exception as e:`, i.e. handling *any* exception.

Something like this would work:

```python
class WordHintingError(Exception):

    """Exception raised on errors during word hinting."""
```

(but also see my comment below, which might make this unnecessary anyways)